tornado/ioloop.py

```diff
@@ -145,12 +144,10 @@ async def main():
        cannot be used on Python 3 except to redundantly specify the `asyncio`
        event loop.
 
-    .. deprecated:: 6.2
-       It is deprecated to create an event loop that is "current" but not
-       running. This means it is deprecated to pass
-       ``make_current=True`` to the ``IOLoop`` constructor, or to create
-       an ``IOLoop`` while no asyncio event loop is running unless
-       ``make_current=False`` is used.
+    .. versionchanged:: 6.3
+       ``make_current=False`` is now the default when creating an IOLoop -
+       previously the default was to make the event loop current if there wasn't
+       already a current one. Passing ``make_current=True`` is deprecated.
```

Comment:
I just noticed that `make_current=True` was already deprecated in 6.2; I can split this out in the docstring if you like.